app/uk/gov/hmrc/plasticpackagingtax/registration/views/check_liability_details_answers_page.scala.html

```diff
@@ -63,13 +61,67 @@
                             )
                         ))
                     ),
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(messages("checkLiabilityDetailsAnswers.future.exceed"))
+                        ),
+                        value = Value(
+                            content = if (preLaunch) {
+                              registration.liabilityDetails.expectToExceedThresholdSoon match {
+                                  case Some(true) => Text(messages("general.true"))
+                                  case Some(false) => Text(messages("general.false"))
+                                  case None => Empty
+                              }
+                            } else {
+                              Empty
+                            }
+                        ),
+                        actions = Some(Actions(
+                            items = Seq(
+                                ActionItem(
+                                    href = pptRoutes.LiabilityStartDateController.displayPage().url,
+                                    content = Text(messages("site.link.change")),
+                                    visuallyHiddenText = Some(messages("checkLiabilityDetailsAnswers.future.exceed"))
+                                )
+                            )
+                        ))
+                    ),
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(messages("checkLiabilityDetailsAnswers.future.liable"))
```

Comment:
So if ` not preLaunch` will it still create the row?
Just being mindful of screenreaders

Reply:
No, it will be stripped in the filter call below: `filterNot(_.value.content == Empty)`